### PR TITLE
Replace "dotnet run" with "dotnet Benchmarks.dll"

### PR DESCRIPTION
**Pull request by mikeharder**
It is recommended to use "dotnet foo.dll" rather than "dotnet run" for production scenarios.  Should not impact throughput.

### CODE REVIEW

File: src/BenchmarksServer/Startup.cs [hunk 1]
```diff
@@ -262,7 +266,8 @@ private static void AddSourceDependencies(string path, string benchmarksDir, IEn
             benchmarksProjectDocument.Root.Add(netFrameworkReferences);
             benchmarksProjectDocument.Root.Add(netCoreReferences);
 
-            using (var writer = XmlWriter.Create(benchmarksProjectPath, new XmlWriterSettings { Indent = true, OmitXmlDeclaration = true }))
+            using (var stream = File.OpenWrite(benchmarksProjectPath))
+            using (var writer = XmlWriter.Create(stream, new XmlWriterSettings { Indent = true, OmitXmlDeclaration = true }))
```
Comment (dougbu) [inline src/BenchmarksServer/Startup.cs]: Curious: Why did you need to manage the underlying stream separately? E.g. does the other `Create()` overload open the file strangely?

Comment (cesarbs) [inline src/BenchmarksServer/Startup.cs]: We don't have the former API now that we've reverted to a previous version of NETStandard.Library.

Comment (mikeharder) [inline src/BenchmarksServer/Startup.cs]: The `Create(string)` overloads only exist in .NET Framework and newer versions of `NETStandard.Library`.  We could go back to the `Create(string)` overload once we move to a newer version of `NETStandard.Library`.
